src/menus/menus.service.ts

```diff
@@ -172,10 +170,10 @@ export class MenusService {
 
   async deleteManyMenus(ids: Types.ObjectId[]) {
     const currentDate = new Date();
-    const deleteManyScript = {
+    const deleteManyAggregation = {
```

Comment:
It's not aggregation. please change to query.

Reply:
k bruh

Reply:
mb